src/main/java/net/fabricmc/loader/lib/gson/JsonWriter.java

```diff
@@ -0,0 +1,662 @@
+/*
+ * Copyright (C) 2010 Google Inc.
+ * Copyright (c) 2020 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
Could you match the additional statement that we added to the end of license header in JsonReader here?

Reply:
Ah I missed that - thanks!

Reply:
Oh, it turns out I based this on an older commit - it's since been changed to exclude the entire package.